.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 package-lock.json
 data
 dist
-.eslintrc.js
\ No newline at end of file
+.eslintrc.js
+.vscode
```

Comment:
I should see .vscode removed from code base
https://stackoverflow.com/questions/936249/how-to-stop-tracking-and-ignore-changes-to-a-file-in-git/19837023